dockerfiles/alpine/Dockerfile

```diff
@@ -26,12 +26,14 @@ RUN echo "$(case "$BUILD_PROFILE" in\
   (*) echo "$BUILD_PROFILE";;\
   esac)" >> ./build_profile_dir
 
-RUN mkdir -p backend/src frontend cli/src utils/src
+RUN mkdir -p backend/src frontend cli/src utils/src proc_macro/src
 RUN mkdir target
 COPY Cargo.toml Cargo.toml
 COPY cli/Cargo.toml cli/Cargo.toml
 COPY backend/Cargo.toml backend/Cargo.toml
 COPY utils/Cargo.toml utils/Cargo.toml
+COPY proc_macro/Cargo.toml proc_macro/Cargo.toml
+COPY proc_macro/src/lib.rs proc_macro/src/lib.rs
```

Comment:
For some strange reason, i need to explicitly copy `proc-macro`'s `lib.rs` file, otherwise the docker build will fail in Github Actions (locally it build just fine)

Reply:
Creating `lib.rs` the same way as at line 42 doesn't work?

Reply:
>Creating lib.rs the same way as at line 42 doesn't work?
 Nope. As far as I understand that, `proc_macro`s in rust depends on some dynamic linkage functions, and afaik "musl" is anything but that, so some strange shenanigans are happening. But I do not know what ** exactly**  kind of shenanigans, so "It just works" is fine, I suppose